cmake/find/datasketches.cmake

```diff
@@ -22,6 +22,8 @@ endif()
 
 if (DATASKETCHES_LIBRARY AND DATASKETCHES_INCLUDE_DIR)
     set(USE_DATASKETCHES 1)
+else()
+    set(USE_DATASKETCHES 0)
```

Comment:
It seems that in's not necessary to write this else. Or I'm wrong?

Reply:
Yes, you are write. I removed the `else` statements, PTAL~